NitroxClient/GameLogic/Spawning/Metadata/Processor/DrillableMetadataProcessor.cs

```diff
@@ -0,0 +1,67 @@
+using System.Linq;
+using Nitrox.Model.Helper;
+using Nitrox.Model.Subnautica.DataStructures.GameLogic.Entities.Metadata;
+using NitroxClient.GameLogic.Spawning.Metadata.Processor.Abstract;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.Spawning.Metadata.Processor;
+
+public class DrillableMetadataProcessor : EntityMetadataProcessor<DrillableMetadata>
+{
+    public override void ProcessMetadata(GameObject gameObject, DrillableMetadata metadata)
+    {
+        Drillable drillable = gameObject.GetComponent<Drillable>();
+        if (!drillable)
+        {
+            Log.Error($"Could not find Drillable on {gameObject.name}");
+            return;
+        }
+
+        drillable.timeLastDrilled = metadata.TimeLastDrilled;
+
+        if (drillable.health == null)
+        {
+            drillable.health = metadata.ChunkHealth;
+            return;
+        }
+
+        Validate.IsTrue(drillable.health.Length == metadata.ChunkHealth.Length);
```

Comment:
I think `Validate.IsTrue` can be dangerous here as depending on the caller the next try catch is far away.